### DISCUSSION TITLE
Server: Siren

### DISCUSSION THREAD

**Discussion by shnabz** [MiguelRipoll23/homebridge-securitysystem#189] [Q&A] [answered]
In my implementation, I do not use automation and the siren to trigger the alarm, instead, I use motion sensors hooked up to NodeRed, which makes a GET request to trigger the alarm.

The only problem is that the trigger delay seconds for home, away and night aren't used. It would be great to activate the siren via the server, instead of calling the triggered function.

**Comment by MiguelRipoll23** [comment, marked as answer]
Hey there,

Have you tried to add the delay parameter like this?

http://localhost:4500/triggered?delay=true

**Reply by shnabz** [reply]
I'm not quite sure how I missed that param... thank you!
